src/rpc/client/ts/tests/fixtures.ts

```diff
@@ -78,24 +81,42 @@ export async function mochaGlobalSetup(): Promise<number | undefined> {
     pid: pid,
     pidfile: getPidFile(testPort),
   })
+
   return pid
 }
 
 /**
  * Mocha test fixture to stop the server
  * @remarks used by mocha
  */
-export function mochaGlobalTeardown(): boolean {
+export async function mochaGlobalTeardown(): Promise<boolean> {
   const pidFile = getPidFile(testPort)
   getLogger().debug({
     fn: 'mochaGlobalTeardown',
     msg: 'stopping server',
     port: testPort,
     pidfile: pidFile,
   })
+
+  if ((await stopServerImmediate()) != 0) {
+    if (fs.existsSync(pidFile)) {
+      fs.unlinkSync(pidFile)
+    }
```

Comment:
If the API method _failed_, then we use the PID method.  This is how we do robust server management using belt and suspenders.

Reply:
We should send a WARN level message that the API method didn't work.